articles/cognitive-services/Face/face-how-to-install-containers.md

```diff
@@ -80,7 +80,7 @@ The container image for the Face container is available from a private Docker co
 
 Use the [docker pull](https://docs.docker.com/engine/reference/commandline/pull/) command to download a container image from the repository. For example, to download the latest Face container image from the repository, use the following command:
 
-```Docker
+```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands